README.md

```diff
@@ -1,15 +1,12 @@
 # vimeo-networking
-vimeo-networking is a Java networking library used for interacting with the Vimeo API. The example app
-provided in this project shows the implementation in the context of an Android app.
+vimeo-networking is a JVM (Java & Kotlin) SDK used for interacting with the Vimeo API. The example app provided in this project shows the implementation in the context of an Android app.
```

Comment:
Yeah, it's an unfortunate name that was chosen. I think we should just leave it though as it might break people's builds for those using JitPack or including the repo as a submodule.